assets/global.js

```diff
@@ -731,16 +739,24 @@ class SlideshowComponent extends SliderComponent {
   }
 
   focusOutHandling(event) {
-    const focusedOnAutoplayButton = event.target === this.sliderAutoplayButton || this.sliderAutoplayButton.contains(event.target);
-    if (!this.autoplayButtonIsSetToPlay || focusedOnAutoplayButton) return;
-    this.play();
+    if (this.sliderAutoplayButton !== undefined ) {
```

Comment:
Same here, we could simply do this I think:
```suggestion
    if (this.sliderAutoplayButton ) {
```